src/Package/Impl/Repl/Commands/ReplShortcutSetting.cs

```diff
@@ -2,91 +2,39 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Diagnostics;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
 using EnvDTE;
-using Microsoft.R.Editor.Settings;
 using Microsoft.VisualStudio.R.Package.Shell;
 
 namespace Microsoft.VisualStudio.R.Package.Repl.Commands {
     internal static class ReplShortcutSetting {
-        private const string CommandName = "EditorContextMenus.CodeWindow.ExecuteLineInInteractive";
-
-        private static bool _currentSetting;
+        private static Dictionary<string, string> _editorShortcutRemaps = new Dictionary<string, string>() {
+            { "Edit.LineOpenAbove", "Text Editor::ctrl+k,ctrl+o" }, // Make Ctrl+Enter work in R Editor
```

Comment:
Apart from Ctrl+Enter and Ctrl+L we only setting shortcuts for our own commands so it doesn't matter what they are - I mean, we are not changing other shortcuts globally. all the shortcuts are set in R Editor, and R Tool windows. Even show solution explorer is out own command so official one in View is unaffected. 4 or 5 combos we specify affect 4-5 obscure commands and only in specific environment. Someone working on C# or C++ won't be affected in any significant way. Also, users familiar with VS can always remap couple of commands in case they are unhappy - they know how to do it. Data folks are not familiar with VS customization on the other hand.